.codesandbox/ci.json

```diff
@@ -1,8 +1,8 @@
 {
-  "buildCommand": "build:codesandbox",
+  "buildCommand": "build",
   "packages": [
-    "handsontable",
     "wrappers/angular",
+    "handsontable",
     "wrappers/react",
     "wrappers/vue"
```

Comment:
WDYT about adding support for Vue3?

Reply:
Yes, I have mentioned this in the original comment - will create another PR. Just want this out the door as the codesandbox integration will be failing on other PRs.